src/utils/wallet/evm.ts

```diff
@@ -14,12 +14,16 @@ import type {
   EvmUnsignedTransaction,
   EvmChains,
 } from '@wormhole-foundation/sdk-evm';
-import type { Network } from '@wormhole-foundation/sdk';
+import type { Chain, Network } from '@wormhole-foundation/sdk';
+import { chainToPlatform } from '@wormhole-foundation/sdk';
 
 import config from 'config';
 import * as ethers from 'ethers';
 import { sleep } from 'utils';
 
+export const isEvmChain = (chain: Chain): boolean =>
```

Comment:
I would move this to a new file evm.ts 1 level above, since this is not wallet related